src/App.tsx

```diff
@@ -1,18 +1,36 @@
-import React from 'react';
+/* eslint-disable react/button-has-type */
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import Pagination from './components/Pagination/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [page, setPage] = useState(1);
+  const [limit, setLimit] = useState(5);
+
+  const getPages = (pages: number = page, limits: number = limit): string[] => {
+    const array = [];
+
+    // eslint-disable-next-line no-plusplus
+    for (let i = (pages - 1) * limits; i < (pages * limits); i++) {
+      array.push(items[i]);
+    }
+
+    return array;
+  };
+
+  const INFO_OF_PAGE = `Page ${page} (items ${(page - 1) * limit + 1} - ${Math.min(page * limit, items.length)} of ${items.length})`;
```

Comment:
It's not stale constant, so use camelCase:
```suggestion
  const infoOfPlace = `Page ${page} (items ${(page - 1) * limit + 1} - ${Math.min(page * limit, items.length)} of ${items.length})`;
```